docs/ldml/tr35-keyboards.md

```diff
@@ -520,207 +595,179 @@ Indicates that:
-
-Within a combination, the presence of a modifier WITHOUT the '?' suffix indicates this key MUST be on. The converse is also true, the absence of a modifier key means it MUST be off for the combination to be active.
-
-Here is an exhaustive list of all possible modifier keys:
-
-###### Table: <a name="Possible_Modifier_Keys" href="#Possible_Modifier_Keys">Possible Modifier Keys</a>
-
-| Modifier Keys |          | Comments                        |
-|---------------|----------|---------------------------------|
-| `altL`        | `altR`   | xAlty → xAltR+AltL? xAltR?AltLy |
-| `ctrlL`       | `ctrlR`  | ditto for Ctrl                  |
-| `shiftL`      | `shiftR` | ditto for Shift                 |
-| `optL`        | `optR`   | ditto for Opt                   |
-| `caps`        |          | Caps Lock                       |
-| `cmd`         |          | Command on the Mac              |
 
-All sets of modifier combinations within a layout are disjoint with no-overlap existing between the key maps. That is, for every possible modifier combination, there is at most a single match within the layout file. There are thus never multiple matches. If no exact match is available, the match falls back to the base map unless the `fallback="omit"` attribute in the `settings` element is set, in which case there would be no output at all.
-
-**Example**
-
-To illustrate, the following example produces an invalid layout because pressing the "Ctrl" modifier key produces an indeterminate result:
-
-```xml
-<keyMap modifiers="ctrl+shift?">
-    …
-</keyMap>
-```
-
-```xml
-<keyMap modifiers="ctrl">
-    …
-</keyMap>
-```
-
-Modifier Examples:
-
-```xml
-<keyMap modifiers="cmd?+opt+caps?+shift" />
-```
-
-Caps-Lock may be ON or OFF, Option must be ON, Shift must be ON and Command may be ON or OFF.
-
-```xml
-<keyMap modifiers="shift caps" />
-```
-
-Caps-Lock must be ON OR Shift must be ON.
-
-If the `modifiers` attribute is not present on a `keyMap` then that particular key map is the base map.
 
 * * *
 
-### 5.9 <a name="Element_map" href="#Element_map">Element: map</a>
-
-This element defines a mapping between the base character and the output for a particular set of active modifier keys. This element must have the `keyMap` element as its parent.
+### 5.9 <a name="Element_key" href="#Element_key">Element: key</a>
 
-If a `map` element for a particular ISO layout position has not been defined then if this key is pressed, no output is produced.
+This element defines a mapping between an abstract key and its output. This element must have the `keys` element as its parent. The `key` element is referened by the `keys=` attribute of the [`row` element](#Element_row).
 
 **Syntax**
 
 ```xml
-<map
- iso="{the iso position}"
- to="{the output}"
+<key
+ id="{key id}"
+ [flicks="{flicks identifier}"]
+ [gap="true"]
```

Comment:
Please contrast against [en-t-k0-teletype.xml](https://github.com/unicode-org/cldr/pull/1847/files#diff-9b1c99aec1ff6f44a068a703a57c0a534183941f44db9128771fff610cc9d0e1) as seen in this PR.

Here, gap is given a boolean type, but in that file, it's numeric.

Reply:
I Didn't get to either the dtd or the other files yet.